industry_restaurant/__manifest__.py

```diff
@@ -0,0 +1,66 @@
+{
+    'name': 'Restaurant (Foods & Cafes)',
+    'version': '1.0',
+    'category': 'Services',
+    'description': """
+        This Odoo module is designed to streamline and enhance the management of your restaurant operations.
+        Whether you own a fine dining establishment, a cafe, or fast-food joints, cafés, food trucks, cloud kitchens, and more.
+    """,
+    'depends': [
+        'account_followup',
+        'knowledge',
+        'pos_enterprise',
+        'pos_loyalty',
+        'sale_management',
+        'pos_online_payment_self_order',
+        'pos_restaurant_appointment',
+        'website_appointment',
+        'website_sale',
+        'payment_demo',
```

Comment:
@vava-odoo Do we really need this module payment_demo? doesn't seems cool when we are not in demo

Reply:
To me, it's a must-have for demo. And I don't think it really pollutes the db if not in demo.
But you're right, it should be unpublished/disabled in data
```xml
<record id="payment.payment_provider_demo" model="payment.provider">
        <field name="is_published" eval="0"/>
        <field name="state">disabled</field>
    </record>
```
and re-enabled in demo. Like we do for other industries

Reply:
Ok, seems a good idea

Reply:
:+1: 
@dhrs-odoo Could you add the missing files?